tests/utils/esx/esxcli.go

```diff
@@ -69,11 +69,11 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	maxAttempt := 20
 	waitTime := 5
 	for attempt := 0; attempt < maxAttempt; attempt++ {
-		misc.SleepForSec(waitTime)
 		status := fn(vmName)
 		if status == expectedState {
 			return true
 		}
+		misc.SleepForSec(waitTime)
```

Comment:
Actually it should be lesser, but I'm not sure why the code is using a 5 second sleep now. I can set it lower but a little over 1.5 minutes seems a long time for a VM to report powered on state by ESX.